doc/reference/algorithms/tree.rst

```diff
@@ -61,6 +61,14 @@ Spanning Trees
    minimum_spanning_edges
    maximum_spanning_edges
 
+Junction Tree Algorithm
```

Comment:
This needs to be fixed as well.

Reply:
I fixed both entries, CircleCI still seems to fail. It won't show me any details on why the build failed, can you help me out here?

Reply:
That was an issue with CircleCI (unrelated to this PR) that I fixed (hopefully) in #4147.  I merged master into your branch and pushed to your remote.  That should fix the CircleCI failure.